### PR TITLE
WIP kitt client

### PR DESCRIPTION
**Pull request by krokrob**
this comes along with kitt product API

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -15,7 +15,7 @@
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
-  create_table "applies", force: :cascade do |t|
+  create_table "applies", id: :serial, force: :cascade do |t|
```
Comment (krokrob) [inline db/schema.rb]: not sure why this `id:` has been added

**Comment by krokrob** [comment]
@ssaunier please review

File: data/projects.yml [hunk 2]
```diff
@@ -0,0 +1,73 @@
+top:
```
Comment (ssaunier) [inline data/projects.yml]: Why `top` ?

File: app/views/projects/index.html.erb [hunk 3]
```diff
@@ -1,9 +1,9 @@
-<%= content_for(:title)       { (t ".meta_title_html", count: @client.projects("alumni_projects").length).html_safe }       %>
+<%= content_for(:title)       { (t ".meta_title_html", count: @kitt_client.products(Static::PROJECTS[:top]).length).html_safe } %>
```
Comment (ssaunier) [inline app/views/projects/index.html.erb]: Maybe you should have this code in the controller:

```ruby
@products = @kitt_client.products(Static::PROJECTS[:top])
```

File: app/services/alumni_client.rb [hunk 4]
```diff
@@ -41,6 +41,7 @@ def projects(list_name = nil)
     end
   end
 
+  # TODO: put back in a yaml in www.
```
Comment (ssaunier) [inline app/services/alumni_client.rb]: 👍

**Comment by ssaunier** [comment]
Don't hesitate to remove methods from `AlumniClient` class as you make progress 👍

**Comment by ssaunier** [comment]
LGTM 👍 
There's only `KITT_WWW_SHARED_SECRET` as a new env key right?

File: data/projects.yml (hunk 2 above)
Comment (krokrob) [inline data/projects.yml]: There are projects we want to promote/show

**Pull request closed without merging**